### ISSUE TITLE
Removal of writer.Flush() method

### ISSUE THREAD

**Issue by harikb** [pierrec/lz4#117]
Hello, 

I see that `Flush()` public method was removed in v4 via this commit
https://github.com/pierrec/lz4/commit/fb4a2ec8e1c77e10b14dda65076589e53c37605b#diff-6326c99dc4638d1cf8582c8ab1941867b252a173a640030c64678793f10a3b5c
Just want to check if this was intentional .  I see that Flush is implied in the Close, but just curious.

If this was intentional, please ignore and close this ticket.

Thanks

**Comment by pierrec** [comment]
Hello,

Yes it was. Do you have any need for it?

**Comment by harikb** [comment]
No it is fine. Anyways Flushing incomplete compressed file isn't helpful anyways. I have worked around it.

As part of a library that provided an interface to write to either Plain/LZ4 (and even gzip in future), I used to have a `.Flush()` method that did the appropriate thing. Although I don't support gz.Writer, that has a Flush method as well. For now i am working around it, so it is not a problem. 

https://github.com/adobe/blackhole/blob/blobstore/lib/archive/common/common.go#L136
